### PR TITLE
Ski packages

### PR DESCRIPTION
**Pull request by paulgrever**

### CODE REVIEW

**Comment by paulgrever** [comment]
@spencern  ready for review!

File: client/templates/navbar/afNavbar.html [hunk 1]
```diff
@@ -98,9 +98,12 @@
...
             <ul class="dropdown-menu">
-              <li><a href="{{pathFor 'missing'}}">Missing Items</a></li>
-              <li><a href="{{pathFor 'damaged'}}">Damaged Items</a></li>
-              <li><a href="{{pathFor 'infoMissing'}}">Orders with Missing Information</a></li>
+              <li><a href="{{pathFor 'missing'}}">Returns with Missing Items</a></li>
+              <li><a href="{{pathFor 'damaged'}}">Returns with Damaged Items</a></li>
+              <li><a href="{{pathFor 'impossibleDates'}}">Impossible Shipping Dates <span class='badge'>{{impossibleDates}}</span></a></li>
+              <li><a href="{{pathFor 'missingRentalDates'}}">Missing Rental Dates <span class="badge">{{missingRentalDates}}</span></a></li>
+              <li><a href="{{pathFor 'missingItemDetails'}}">Orders with Missing Item Details <span class='badge'>{{missingItemsFromOrder}}</span></a></li>
+              <li><a href="{{pathFor 'missingBundleColors'}}">Orders with Missing Bundle Colors <span class='badge'>{{missingBundleInfo}}</span></a></li>
```
Comment (spencern) [inline client/templates/navbar/afNavbar.html:106]: Love this feature!

File: server/methods/customerService.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }, {
+      $addToSet: {
+        history: history
+      },
+      $set: {
+        bundleMissingColor: false
+      }
+    });
+  },
+  'advancedFulfillment/updateSkiPackageWithCustomerInfo': function (orderId, userId, skiId, age, shoeSize, level) {
```
Comment (spencern) [inline server/methods/customerService.js:41]: We do need to start making sure we are checking for proper roles in these methods - especially these methods where we are taking params and putting them directly into the database.

File: server/methods/orderDetail.js [hunk 3]
```diff
@@ -274,5 +248,196 @@ Meteor.methods({
         }
       }
     });
+  },
+  'advancedFulfillment/updateRentalDates': function (orderId, startDate, endDate) {
```
Comment (spencern) [inline server/methods/orderDetail.js:252]: Same concerns here with security.

**Pull request merged**